go.mod

```diff
@@ -1,29 +1,31 @@
 module github.com/segmentio/stats/v4
 
-go 1.22.0
+go 1.23.0
+
+toolchain go1.24.4
```

Comment:
Can we remove this line? This forces a redownload of the entire Go binary for anyone who is running this.

Reply:
Definitely.

Reply:
Fixed in af8303c